packages/cli/generators/datasource/index.js

```diff
@@ -134,7 +134,7 @@ module.exports = class DataSourceGenerator extends ArtifactGenerator {
       const prompts = [
         {
           name: 'customConnector',
-          message: "Enter the connector's module name",
+          message: "Enter the connector's module name:",
```

Comment:
Would it be better to emphasize "enter the FULL name"?
I remember in lb3 w/o the prefix `loopback-connector` confused people.